sw/device/tests/sim_dv/flash_rma_unlocked_test.c

```diff
@@ -0,0 +1,315 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/arch/device.h"
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/dif/dif_flash_ctrl.h"
+#include "sw/device/lib/dif/dif_lc_ctrl.h"
+#include "sw/device/lib/dif/dif_otp_ctrl.h"
+#include "sw/device/lib/dif/dif_uart.h"
+#include "sw/device/lib/pinmux.h"
+#include "sw/device/lib/runtime/hart.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/runtime/print.h"
+#include "sw/device/lib/testing/check.h"
+#include "sw/device/lib/testing/flash_ctrl_testutils.h"
+#include "sw/device/lib/testing/test_framework/test_status.h"
+
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"  // Generated.
+
+#define LC_TOKEN_SIZE 16
+
+static dif_uart_t uart0;
+static dif_flash_ctrl_state_t flash_state;
+static dif_lc_ctrl_t lc;
+
+// TODO : GitHub Issue #11795
+// Using retention SRAM to record the test_phase. When the
+// sw_symbol_backdoor_overwrite is fixed, this can be overriden by
+// the testbench as a SW variable.
```

Comment:
nit:
```suggestion
// TODO(lowRISC/opentitan:#11795): when the sw_symbol_backdoor_overwrite is fixed for ROM, this can be overriden by the testbench as a SW variable.
```

Reply:
style applies throughout